.github/workflows/api-tests.yaml

```diff
@@ -0,0 +1,82 @@
+name: API Testing
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+    branches:
+      - main
+    paths:
+      - "api/**"
+      - ".github/workflows/api-tests.yaml"
+  workflow_dispatch:
+
+jobs:
+  integration-tests:
+    name: API Integration Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Install Dependencies
+        working-directory: ./api
+        run: |
+          npm i
+
+      # Ups the rate limiter max to an unreasonable number. Avoids 429 responses.
+      - name: Relax Express Rate Limiter
+        working-directory: ./api
+        run: |
+          sed -i 's/max: [0-9]\+,/max: 99999,/g' express.ts
+
+      # ENV values match local development seed values only.
+      # Does not allow for Keycloak or GC Notify tests
+      - name: Create Mock ENV
+        run: |
+          touch .env
```

Comment:
Unless you're specifically relying on a .env file, you can set env variables at the workflow, job, or step level.

Reply:
Interesting. I'll look into it.

Reply:
Changed this out for env spec for the whole workflow.